Gruntfile.js

```diff
@@ -1,5 +1,13 @@
 'use strict';
 
+var path = require('path');
+var config = require('config');
+
+let dataDirectory = config.get('database.data_directory');
+if (dataDirectory === '') {
```

Comment:
It won't throw an error, because it does exist. The default is `''`, which uses the project directory.